typescript-deno-plugin/src/plugin.ts

```diff
@@ -224,12 +224,7 @@ export class DenoPlugin implements ts_module.server.PluginModule {
         }
 
         const importMapsFilepath = this.configurationManager.config.import_map
-          ? path.isAbsolute(this.configurationManager.config.import_map)
-            ? this.configurationManager.config.import_map
-            : path.resolve(
-                project.getCurrentDirectory(),
-                this.configurationManager.config.import_map
-              )
+          ? this.configurationManager.config.import_map
```

Comment:
Missing the config update in typescript-deno-plugin


https://github.com/denoland/vscode_deno/blob/4e23f8c7e3922d360eb6fa75c885c34c4538a250/typescript-deno-plugin/src/plugin.ts#L557-L559

Reply:
@axetroy I checked the file and it seems that the only place that reads `DENO_DIR` is https://github.com/denoland/vscode_deno/pull/166/files#diff-ea9fa7070d48e644600141672d50fb9cR189-R207, so I have added the config update there. Please correct me if I am wrong.